src/bgfx.cpp

```diff
@@ -507,6 +510,7 @@ namespace bgfx
 			}
 		}
 
+		BX_CHECK(false, "Failed to create embedded shader");
```

Comment:
Remove this, because it provides invalid shader as signal that it failed to create shader.

Reply:
Replaced with a check when creating embedded shaders for blitter and clear, since that was were there was a lack of reporting